public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -122,15 +121,34 @@ function AdminGSVLabelView(admin, source) {
                                     '<tr>' +
                                         '<th>Previous Validations</th>' +
                                         '<td id="prev-validations"></td>' +
-                                    '</tr>'
+                                    '</tr>';
         }
-        modalText +=
-                                '</table>' +
-                            '</div>' +
+        modalText +=            '</table>';
+        if (self.admin) {
+            modalText +=        '<div id="flag-input-holder">' +
+                                    `<h3 style="margin: 0px; padding-top: 10px;">Manually set work quality for street</h3>` +
```

Comment:
Can you try to put most of the inline CSS here into CSS classes in a css file? The code here is already gross (not your fault), so it's not like everything needs to be moved out. But I'm assuming that there are at least a couple CSS classes that could be created that would make this look more reasonable.